static/styles/subscriptions.scss

```diff
@@ -1032,7 +1046,7 @@ ul.grey-box {
     }
 }
 
-@media (max-width: 700px) {
+@media (max-width: 790px) {
```

Comment:
Why this specific change?

Reply:
This prevents the model from splitting to into side-by-side view for another 90 pixels, and makes the search boxes fit for the 700-790 px range.

Reply:
Keeping it at 700px did break the UI. This works great.